yellowbrick/text/dispersion.py

```diff
@@ -58,19 +71,30 @@ class DispersionPlot(TextVisualizer):
     process, but can and should be set as early as possible.
     """
 
-    def __init__(self, words, ax=None, color=None, ignore_case=False,
-                 annotate_docs=False, **kwargs):
+    # NOTE: cannot be np.nan
+    NULL_CLASS = None
+
+    def __init__(self, words, ax=None, colors=None, ignore_case=False,
+                 annotate_docs=False, labels=None, colormap=None, **kwargs):
         super(DispersionPlot, self).__init__(ax=ax, **kwargs)
 
-        self.color = color
+        self.labels = labels
+        self.colors = colors
+        self.colormap = colormap
+
         self.words = words
         self.ignore_case = ignore_case
         self.annotate_docs = annotate_docs
 
-    def _compute_dispersion(self, text):
+    def _compute_dispersion(self, text, y):
         self.boundaries_ = []
         self.offset = 0
```

Comment:
Does `self.offset` need to be a class attribute? I looked but it doesn't appear to be used outside of the `_compute_dispersion` function.

Reply:
I agree!